### ISSUE TITLE
2020-09-24 How to set up a Windward server on GCP and some insight into systemd

### ISSUE THREAD

**Issue by kdungs** [kdungs/dun.gs#10]
Discussion thread on the article "How to set up a Windward server on GCP and some insight into systemd" posted by Kevin on September 24, 2020.

**Comment by furtivesage** [comment]
Hello kdungs,

I am not sure I am following this section of your article.  Am I creating a directory path and then using Nano to create a windward.service file? What is the entire path where the windward.service is located?  

I have had little experience with this procedure.  Thank you.

Then, create ~/.config/systemd/user/windward.service with the following contents

[Unit]
Description=Windward Dedicated Server

[Service]
Type=oneshot
RemainAfterExit=yes
ExecStart=/usr/bin/tmux new-session -d -s Windward %h/start-server.sh
ExecStop=/usr/bin/tmux send-keys -t Windward q Enter Enter

[Install]
WantedBy=default.target
Enable and start the service via

systemctl --user enable windward.service
systemctl --user start windward.service

**Comment by kdungs** [comment]
Hi! Cool that you found my post useful. The full path is `~/.config/systemd/user/windward.service`. On Linux, `~` is a shorthand for the current user's home directory. So starting from your home, the relative path is `.config/systemd/user/windward.service`.

Does that help?

**Comment by furtivesage** [comment]
Yes, I will give that a try. Thank you for responding, I greatly appreciate
it.

Best Regards,

Ryan
